docker/root/init.php

```diff
@@ -3,6 +3,10 @@
 if (count(scandir('/hleb')) == 2) {
     exec('composer create-project phphleb/hleb /hleb');
     copy('/hleb/default.start.hleb.php', '/hleb/start.hleb.php');
+    setDbEnvParams('MYSQL_DATABASE_PORT');
+    setDbEnvParams('MYSQL_DATABASE');
+    setDbEnvParams('MYSQL_USER');
+    setDbEnvParams('MYSQL_PASSWORD');
     rename('/root/dbase.config.php', '/hleb/database/dbase.config.php');
```

Comment:
```suggestion
    installDbConfig();
```
Всё, что касается установки конфига СУБД, можно поместить в одну функцию. Во-первых, меньше обращений к файловой системе, а во-вторых, на мой взгляд, так и читается легче